File: Java Core Programming/if else/Quadratic.java
/*WAP to find the roots of quaratic equation ax2 +bx +c=0*/

import java.util.Scanner;

class Quadratic
{
	public static void main(String [] args)
	{
	int r1,r2,sum;
	Scanner sc=new Scanner(System.in);
	System.out.print("Enter the Value of a,b,c of the Quadratic Equation: ");
	int a=sc.nextInt();
	int b=sc.nextInt();
	int c=sc.nextInt();

	sum=(b*b)-4*a*c;

	  if(sum<0)
	  {
	 System.out.println("\nRoots are Imaginary OR Roots are not Possible");
	  }
	  else
	  {
	  r1=(-b+sum)/(2*a);
	  r2=(-b-sum)/(2*a);
	 System.out.println("\nRoots are "+r1+" "+r2);
	  }
	}
}

File: C Programming/Quadratic.cpp
/*WAP to find the roots of quaratic equation ax2 +bx +c=0*/
#include <stdio.h>
#include<conio.h>
#include<maths.h>
void main()
{
clrscr();
int b ,a,c ,r1,r2,sum;
printf("Enter the value of a,b,c of the quaratic equation\n");
scanf("%d%d%d",&a,&b,&c);
sum=(b*b)-4*a*c;
if(sum<0)
{
printf("Roots are not possible");
}
else
{
r1=(-b+sum)/(2*a);
r2=(-b-sum)/(2*a);
printf("Roots are %d, %d",r1,r2);
}
getch();
}